models/app/features/panels/line-plot/reference.mdx

```diff
@@ -2,99 +2,145 @@
 title: Line plot reference
 ---
 
-## X-Axis
+This page provides comprehensive details for line plot settings. For more details about working with line plots, see the [Line plots overview](/models/app/features/panels/line-plot).
+
+## Data settings
+
+### X-axis
 
 <Frame>
     <img src="/images/app_ui/reference_x_axis.png" alt="Selecting X-Axis"  />
 </Frame>
 
-You can set the x-axis of a line plot to any value that you have logged with W&B.log as long as it's always logged as a number.
+Set the x-axis to any value logged with `wandb.log()` as a number.
```

Comment:
What does ...as a number." mean? I think an example and/or word smithing might help. 

We can rewrite this as "Set the coordinates of the x-axis to any value logged with `wandb.Run.log()`. Or "Set the range of the x-axis to any integer or float value you logged with `wandb.Run.log()`. For example, ...."